activerecord/CHANGELOG.md

```diff
@@ -689,4 +689,29 @@
 
     *Myles Boone*
 
+*   Add support for exact attribute filtering in ParameterFilter
```

Comment:
New changelog entries go to top of the file.

Reply:
Ahhhh I see 🤦 Let me fix that!